qa/L0_quick_start/test.sh

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RET=0
+
+model-analyzer profile \
+  -m /quick_start_repository/ \
+  --profile-models add_sub \
+  --run-config-search-max-concurrency 2 \
+  --run-config-search-max-instance-count 2 \
+  --run-config-search-preferred-batch-size-disable true ; \
+  if [ $? -ne 0 ] ; then RET=1 ; fi
+
+mkdir analysis_results ; \
+  if [ $? -ne 0 ] ; then RET=1 ; fi
+
+model-analyzer analyze \
+  --analysis-models add_sub \
+  -e analysis_results ; \
+  if [ $? -ne 0 ] ; then RET=1 ; fi
```

Comment:
Lets have the return value accumulate. This way we know at a glance, how poorly the test performed.

Reply:
discussed offline--updated code to exit early in these cases